.travis.yml

```diff
@@ -0,0 +1,18 @@
+script:
+  # By default Travis CI executes './gradlew build connectedCheck' if no 'script:' section found.
+  - ./gradlew build
+
+language: android
+
+jdk:
+  - oraclejdk8
+
+android:
+  components:
+    - tools
+    - build-tools-23.0.3
+    - android-23
+    - extra-android-m2repository
+
+#notifications:

```

Comment:
It seems like e-mails are sent automatically to the GitHub organization owner (info@literacyapp.org) on each build when `notifications` is missing, as as default notification mechanism. So update to the following:

```
notifications:
  email: false
```